src/deepsparse/yolact/schemas.py

```diff
@@ -67,6 +68,33 @@ class YOLACTInputSchema(ComputerVisionSchema, Splittable):
         description="Confidence threshold applied to the raw detection at "
         "`postprocess` step (optional)",
     )
+    return_masks: bool = Field(
+        default=False,
+        description="Controls whether the pipeline should additionally "
+        "return segmentation masks",
+    )
+
+    @classmethod
+    def from_files(cls, files: Iterable[TextIO], **kwargs) -> "YOLACTInputSchema":
+        """
+        :param files: Iterable of file pointers to create YOLACTInput from
+        :param kwargs: extra keyword args to pass to YOLACTInput constructor
+        :return: YOLACTInput constructed from files
+        """
+        if "images" in kwargs:
+            raise ValueError(
+                f"argument 'images' cannot be specified in {cls.__name__} when "
+                "constructing from file(s)"
+            )
+        files_numpy = [numpy.array(Image.open(file)) for file in files]
+        input_schema = cls(
+            # if the input comes through the client-server communication
+            # do not return segmentation masks
+            images=files_numpy,
+            return_masks=not kwargs.get("from_server"),
```

Comment:
```suggestion
            return_masks=not kwargs.pop("from_server", False),
```

Reply:
We might get `TypeError: ... got multiple values for argument ....` if not popped